client/src/components/Shipping/ShippingOptions.jsx

```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+import React from 'react';
+
+function ShippingOptions({ details }) {
+  if (details && details[0].fastShipping) {
+    var newDate = moment().add(2, 'day').format('dddd, MMMM Do');
+  }
+  return (
+    <div className="shippingOptions">
+      <h3>Shipping options</h3>
+      <p>
+Get it by
```

Comment:
nit: indentation issue?

Reply:
I think that was how eslint suggested it, but I fixed this to make it more readable.